egs/wsj/s5/steps/libs/nnet3/train/chain_objf/acoustic_model.py

```diff
@@ -424,6 +424,19 @@ def prepare_initial_acoustic_model(dir, run_opts, srand=-1):
     common_train_lib.prepare_initial_network(dir, run_opts,
                                              srand=srand)
 
+    # edits 0.raw using edits.config before adding transition model.
+    edits_config_file = "{0}/configs/edits.config".format(dir)
+    if os.path.exists(edits_config_file):
+        logger.info("edits 0.raw model using {0}/configs/edits.config."
+                    "".format(dir))
+        common_lib.execute_command(
+            """{command} {dir}/log/edit.log \
+                nnet3-copy --edits-config={edits_config} {dir}/0.raw \
```

Comment:
I suggest you move this inside prepare_initial_network as this is not just for chain objf, but generic for anything. 
Also you are modifying 0.raw itself. Its better to write an intermediate model like pre_edit.raw.